services/src/registry.ts

```diff
@@ -180,7 +180,7 @@ interface PolicyInput {
   query?: PolicyQueryInput;
 }
 
-const resourceRepository = S3ResourceRepository.fromEnvironment();
+const resourceRepository = S3ResourceRepository.fromEnvironment({ isRegistry: true });
```

Comment:
`isRegistry` flag looks very general. Maybe we can use specific flag (`disablePoliciesLoading` or something like this). This flag can be initialized in `env-var` to default `false` and set in `registry` to `true`.

Reply:
I don't see why we would want a specific flag here, it just hides the reason we decided to do it (which is that the code runs in the registry).

If we don't separate code between the registry and the gateway we do need a way to tell between them when necessary.
I prefer letting each function decide the necessary logic based on the information it needs (whether we are in the registry or not) than the registry and gateway having intimate knowledge about each feature and deciding which flags to send for every feature.